### ISSUE TITLE
Firefox 60.x crashes

### ISSUE THREAD

**Issue by jonbjk** [guybedford/es-module-shims#67]
When loading es-module-shims on older Firefox (60.9.0esr) I get the following error:
```
ReferenceError: can't access lexical declaration `hasDocument' before initialization
```
I think the problem is that 'hasDocument' is initialized after the the try-catch block that is supposed to fix dynamic imports for older Firefox:
https://github.com/guybedford/es-module-shims/blob/master/src/common.js#L16-L40

**Comment by guybedford** [comment]
Ah nice catch, thanks for posting.

**Comment by jonbjk** [comment]
Also saw in the produced file in /dist/es-module-shims.js that pageBaseUrl is not defined, so there seem to be something wrong in the build scripts.

**Comment by guybedford** [comment]
Thanks for these - I've published 0.4.5 with the fixes.

**Comment by jonbjk** [comment]
That was quick!
Thank you for a very good library
